### ISSUE TITLE
gvr_gesture.h contains typos

### ISSUE THREAD

**Issue by shiena** [googlevr/gvr-android-sdk#544]
```diff
diff --git a/libraries/headers/vr/gvr/capi/include/gvr_gesture.h b/libraries/headers/vr/gvr/capi/include/gvr_gesture.h
index 5a28880..e2d1ac8 100644
--- a/libraries/headers/vr/gvr/capi/include/gvr_gesture.h
+++ b/libraries/headers/vr/gvr/capi/include/gvr_gesture.h
@@ -64,13 +64,13 @@ extern "C" {
 ///         case GVR_GESTURE_SWIPE:
 ///           // Handle swipe gesture.
 ///           break;
-///         case GVR_GESTURE__SCROLL_START:
+///         case GVR_GESTURE_SCROLL_START:
 ///           // Handle the start of a sequence of scroll gestures.
 ///           break;
-///         case GVR_GESTURE__SCROLL_UPDATE:
+///         case GVR_GESTURE_SCROLL_UPDATE:
 ///           // Handle an update in a sequence of scroll gestures.
 ///           break;
-///         case GVR_GESTURE__SCROLL_END:
+///         case GVR_GESTURE_SCROLL_END:
 ///           // Handle the end of a sequence of scroll gestures.
 ///           break;
 ///         default:
@@ -243,13 +243,13 @@ typedef gvr_gesture Gesture;
 ///         case GVR_GESTURE_SWIPE:
 ///           // Handle swipe gesture.
 ///           break;
-///         case GVR_GESTURE__SCROLL_START:
+///         case GVR_GESTURE_SCROLL_START:
 ///           // Handle the start of a sequence of scroll gestures.
 ///           break;
-///         case GVR_GESTURE__SCROLL_UPDATE:
+///         case GVR_GESTURE_SCROLL_UPDATE:
 ///           // Handle an update in a sequence of scroll gestures.
 ///           break;
-///         case GVR_GESTURE__SCROLL_END:
+///         case GVR_GESTURE_SCROLL_END:
 ///           // Handle the end of a sequence of scroll gestures.
 ///           break;
 ///         default:
```

These documents also have the same typos.
https://developers.google.com/vr/reference/android-ndk/group/gesture
https://developers.google.com/vr/reference/android-ndk/class/gvr/gesture-api

**Comment by jdduke** [comment]
Good catch, we'll try to get this fixed in the next release.

**Comment by jdduke** [comment]
Expect a doc fix in the 1.50 release. Thanks again for the report.
